InclusionBot.md

```diff
@@ -73,6 +73,8 @@ approves your changes, they will be merged and become active in the bot.
 | master/slave, master / slave                     | primary/secondary<br>primary/replica<br>leader/worker<br>main/replica                         | This term is rooted in the enslavement of human beings. We want to avoid slavery references.<br><br>See [GitHub to replace "master" with alternative term to avoid slavery references](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/) for more.                                                                                                            |
 | open the kimono, open kimono                     | share information                                                                             | This term arises from the sexualization of Asian women. We want to avoid sexist, racist, and sexual references.<br><br>See [The Problematic Origins of Common Business Jargon](https://www.fastcompany.com/90239290/six-business-phrases-that-have-racist-origins) for more.                                                                                                                                           |
 | grandfathered                                    | legacy<br>included retroactively                                                              | This term arose from voter restrictions that prevented Black people from voting. We want to avoid slavery and racist oppression references.<br><br>See [The Problematic Origins of Common Business Jargon](https://www.fastcompany.com/90239290/six-business-phrases-that-have-racist-origins) for more.                                                                                                               |
+| Goddamn, goddam, goddamnit, goddammit            | awful, terrible, wretched                                                                     | This term implies that God condemns, curses or reproaches. We want to show respect towards all religious traditions.more.                                                                                                                                                                                                                                                                                              |
```

Comment:
I think we should workshop the wording on the explanation because as-written, it reads to me that it endorses/accepts the existence of a single deity (capital-G "God"), which we as civil servants probably shouldn't do? What about something like:

> This term can be offensive or triggering to some religious traditions because it implies a deity cursing or reproaching people. We want to show respect towards all religious traditions.

Reply:
Thanks, LGTM. I move it to the source doc so other people can weight in.

Reply:
@abrouilette waiting for others to weight in on the document

Reply:
Enough weighing-in time has passed.  Make it so, sir!